src/main/java/emissary/server/api/Version.java

```diff
@@ -81,4 +87,36 @@ private MapResponseEntity lookupVersion() {
         return entity;
     }
 
+    @GET
+    @Path("/versionSimple")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response versionSimple() {
+        MapResponseEntity entity = new MapResponseEntity();
+        try {
+            GitRepositoryState gitRepoState = GitRepositoryState.getRepositoryState();
+            entity.addKeyValue("Version", gitRepoState.getBuildVersion());
```

Comment:
The new response does not return the host:port of the server. I figured if the endpoint is being hit then then the user already knows the server name and port and it does not need to be returned. Also not sure how to get those values :)

Reply:
Why would the response need those? (unless the response is being teed directly into a log file and we want those to be pat of the logged json payload)